packages/uniwind/tests/test.css

```
@@ -10,4 +10,17 @@
   --color-foo: hsl(var(--foo));
   --color-bar: oklch(var(--bar));
   --spacing-test: 123px;
+  --color-background: unset;
+}
+
+@layer theme {
+    :root {
+        @variant dark {
```

Stylelint (17.3.0)

[error] Unexpected unknown at-rule "@variant" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)